src/conversions/num_complex.rs

```diff
@@ -152,6 +152,19 @@ macro_rules! complex_conversion {
 
                 #[cfg(any(Py_LIMITED_API, PyPy))]
                 unsafe {
+                    use crate::{intern, type_object::PyTypeInfo};
+
+                    let obj = if PyComplex::is_type_of(obj) {
```

Comment:
Purely a style question, I think `obj.is_instance_of::<PyComplex>()` would be easier to read.

Reply:
No problem - changed in b15573f2.